source/_components/sensor.uscis.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: USCIS Sensor
+description: "Instructions on how to set up USCIS within Home Assistant."
+date: 2018-03-21 08:00
+sidebar: false
+comments: false
+sharing: true
+footer: true
+logo: uscis.jpg
+ha_category: Sensor
+ha_release: 0.68
+ha_iot_class: "Cloud Polling"
+---
+
+The `uscis` sensor component allows you get updates on you USCIS case using your case number/ receipt number
```

Comment:
You may want to open what is USCIS a bit in the description, at least by adding a link there.

* "on your USCIS case"
* "case number / receipt number" (whitespace)